email_slicer_using_Cli_handler.py

```diff
@@ -0,0 +1,32 @@
+class Email_slicer:
+    def __init__(self, email):
+        self._email = email
+
+    def is_valid_email(self) -> bool:
+        return '@' not in self._email or self._email[len(self._email) - 4:] != '.com'
+
+    def username(self):
+        return self._email[:self._email.index('@')]
+
+    def domain_name(self):
+        return self._email[self._email.index('@') + 1:]
+
+
+class Cli_handler:
+    def __init__(self, email):
+        self._email = email
```

Comment:
No need of `self._email` variable as this is not used anywhere in the other methods of this class.